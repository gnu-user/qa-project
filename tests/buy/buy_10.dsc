TITLE
    Ticket Purchase Number (more than 4, full-standard)
DESCRIPTION
    Run the login transaction using a full-standard account. Initiate the buy 
    transaction. When prompted, enter a title which exists. When prompted, 
    enter a number of tickets to purchase which is greater than 4. Verify that 
    the amount is not accepted, and an error occurs.
DEPENDENCIES
    buy_8
