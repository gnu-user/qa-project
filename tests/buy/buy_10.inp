login
fulluser
buy
The Godfather
5