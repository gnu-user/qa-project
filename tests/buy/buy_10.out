Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the event title: 
Enter the number of tickets to purchase: 
Invalid number of tickets, cannot purchase more than 4 tickets.
> 